form.js

```diff
@@ -0,0 +1,13 @@
+const form = document.querySelector('.form');
+const emailError = document.querySelector('.form-error');
+
+form.addEventListener('submit', (e) => {
+  const firstName = form.name.value;
+  const lastName = form.name.value;
+  const email = form.email.value;
+  const message = form.message.value;
+  if (email !== email.toLowerCase()) {
+    e.preventDefault();
+    emailError.innerHTML = `<p class="error">Email should be written in lowercase letters: ${email}</p>`;
```

Comment:
- [ ] Your validation works and the error message appears on screen but it is not appearing in the correct position. Kindly fix the style of the error message.
 - Mobile view:
 - 
<img width="480" alt="image" src="https://user-images.githubusercontent.com/61919617/196970746-c1ce827e-3754-40e8-851c-27d9ffff4cfd.png">
 - Desktop view:
 - 
<img width="1364" alt="image" src="https://user-images.githubusercontent.com/61919617/196971076-f592d49b-9705-4dc9-8ca9-2893c9c1941b.png">